src/shared/Logo/ui/index.tsx

```diff
@@ -0,0 +1,28 @@
+import { LOGO_URL } from "../lib/constants"
+
+export interface LogoProps {
+  widthPx?: number
+  heightPx?: number
```

Comment:
Please define a separate interface `Dimension` with properties: `width` and `height` and use it here like this:
```
export interface LogoProps {
    size: Dimension;
    ...
}
```

Reply:
Done